scout/server/extensions/chanjo2_extension.py

```diff
@@ -44,3 +47,39 @@ def mt_coverage_stats(self, individuals: dict) -> Dict[str, dict]:
             coverage_stats[ind["individual_id"]] = coverage_info
 
         return coverage_stats
+
+    def get_gene_complete_coverage(
+        self, gene: int, threshold: int = 15, individuals: dict = {}, build: str = "38"
+    ) -> bool:
+        """
+        Return complete coverage for hgnc_id at a coverage threshold.
+        """
+        chanjo_build = CHANJO_BUILD_37 if "37" in build else CHANJO_BUILD_38
+        chanjo2_gene_cov_url: str = "/".join(
+            [current_app.config.get("CHANJO2_URL"), "coverage/d4/genes/summary"]
+        )
+
+        gene_cov_query = {
+            "build": chanjo_build,
+            "coverage_threshold": threshold,
+            "hgnc_gene_ids": [gene],
```

Comment:
Same here. I would rename this to hgnc_id. Just because you could provide also hgnc symbol or Ensembl IDs to chanjo2